app/assets/stylesheets/application.css.scss

```diff
@@ -31,4 +31,8 @@
 
 .card_remove_image {
   margin-left: 20px;
+}
+
+.block_user_current_block_id {
```

Comment:
Selector `block_user_current_block_id` should be written in lowercase with hyphens